frontend/public/components/api-explorer.tsx

```diff
@@ -644,6 +645,19 @@ const APIResourcePage_ = ({
     },
   ];
 
+  const namespace = kindObj.namespaced ? match.params.ns : null;
+
+  const canCreate = useAccessReview({
```

Comment:
This name sounds like you're checking if you can create the resource, not the access review.

```suggestion
  const canCreateResourceAccessReview = useAccessReview({
```